guillotina/contrib/pubsub/__init__.py

```diff
@@ -0,0 +1,16 @@
+
+app_settings = {
```

Comment:
Should we add dependency of `guillotina.contrib.cache`?

`"applications": ["guillotina.contrib.cache"]`?